Kernel/FileSystem/FATFS/Definitions.h

```diff
@@ -39,9 +72,22 @@ struct [[gnu::packed]] FAT32BootRecord {
     u8 signature;
     u32 volume_id;
     char volume_label_string[11];
-    char system_identifier_string[8];
+    char file_system_type[8];
+};
+static_assert(sizeof(DOS7BIOSParameterBlock) == 54);
```

Comment:
We have a handy dandy AssertSize helper for this purpose that is preferred

Reply:
Added in new commit 9efb28194f04f954b91745d44e2bd4ee51d052f6, because I updated all structs in Definitions.h (including those untouched by this change).